keyboards/anavi/knob1/keymaps/default/keymap.c

```diff
@@ -0,0 +1,43 @@
+// Copyright 2022 Leon Anavi <leon@anavi.org>
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+
+#include <stdio.h>
```

Comment:
```suggestion
```

Reply:
Please note this is required to avoid the following error:

```
./keyboards/anavi/knob1/keymaps/default/keymap.c:35:3: error: incompatible implicit declaration of built-in function 'snprintf' [-Werror]
./keyboards/anavi/knob1/keymaps/default/keymap.c:5:1: note: include '<stdio.h>' or provide a declaration of 'snprintf'
```